### PR TITLE
Feature/backfill homer for claims with no transcription

### PR DESCRIPTION
**Pull request by meletios**
# Jira Issue: []

## What?
Backfill claims with transcript

## Why?
- Why not! We are going to get more transcripts

## Optional checklist
- [ ] Codescouted
- [ ] Unit tests written
- [ ] Tested locally

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -265,6 +265,7 @@
     <dependency>
       <groupId>com.google.cloud</groupId>
       <artifactId>google-cloud-speech</artifactId>
+      <version>1.24.11</version>
```
Comment (Michael-Duivestein) [inline pom.xml:268]: This dependency is managed by Google's BOM. 
https://github.com/HedvigInsurance/claims-service/pull/81 Updates Google's BOM from 5.4.0 to 16.4.0. This version contains google-cloud-speech 1.24.11: https://github.com/googleapis/java-speech

**Review by Michael-Duivestein — APPROVED** [review]

File: src/main/java/com/hedvig/claims/aggregates/ClaimsAggregate.java [hunk 2]
```diff
@@ -358,7 +358,7 @@ public void on(SetClaimFileCategoryCommand cmd) {
     }
 
     @CommandHandler
-    public void on(AudioTranscribedCommand cmd) {
+    public void on(TranscribeAudioCommand cmd) {
```
Comment (AlexSJ1990) [inline src/main/java/com/hedvig/claims/aggregates/ClaimsAggregate.java:361]: 🧹

**Review by AlexSJ1990 — APPROVED** [review]
Looks good!

File: src/main/java/com/hedvig/claims/serviceIntegration/predictor/PredictionResponse.kt [hunk 3]
```diff
@@ -0,0 +1,5 @@
+package com.hedvig.claims.serviceIntegration.predictor
+
+data class PredictionResponse(
+    val response: String
+)
```
Comment (vonElfvin) [inline src/main/java/com/hedvig/claims/serviceIntegration/predictor/PredictionResponse.kt:5]: Would be nice to get the confidence level as well, but depending on the model I'd see how that could be hard

**Pull request merged**